traitsui/editor.py

```diff
@@ -552,8 +551,10 @@ def _bind_from(self, key, user_object, xuser_name, editor_name, is_list):
         key : str
             The key to use to guard against recursive updates.
         user_object : object
+            The object in the TraitsUI context that is being bound.
+        xuser_name: : str
             The extended name of the trait to be used on the user object.
-        editor_name : string
+        editor_name : str
```

Comment:
There are other uses of `string` as type descriptions in this module, but changing all those back to `str` in this PR would be feature creep, I think.

Reply:
Fixed in #686